demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -547,6 +548,23 @@ def is_changed_context_path(self):
 
         return False
 
+    def is_removed_integration_parameters(self):
```

Comment:
shouldn't this be already covered by the case that that a required parameter was changed? do we really need a new function for this?

Reply:
`is_added_required_fields` Is for adding new required params.
I thought of putting it in the same func, but IMO it is best suited in a different one.